tests/flow/test_slowlog.py

```
@@ -108,3 +110,166 @@ def test02_slowlog_reset(self):
+        # reported param should be truncated
+        self.env.assertIn("...", params)
+        self.env.assertLess(len(params), len(long_string))
+
+    def test04_same_query_diff_params(self):
+        # make sure no new entries are added when the query remains the same
+        # but the params change
+
+        # clear slowlog
+        self.redis_con.execute_command("GRAPH.SLOWLOG", GRAPH_ID, "RESET")
+
+        query = f"UNWIND range(0, $i) AS x RETURN count(x)"
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)